### PR TITLE
Add Inverse + Copy Button

### PR DESCRIPTION
**Pull request by Spoofardio**
Hi! I was doing this math in a CLI tool I was making but came across your site and repo! I see you havn't made changes in a while but figured I'd make a PR as I'm gonna use this locally anyway :). I Need to double check the math though! Very nice repo!
new section

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
     "private": true,
     "version": "0.4.1",
     "engines": {
-        "node": "12.18.3",
-        "yarn": "1.22.4"
+        "node": "16.20.2",
+        "yarn": "1.22.21"
```
Comment (damonhook) [inline package.json:7]: Will also have to update the `.github/workflows/nodejs.yaml` to use the new versions of node and yarn

Line 8
```yaml
node-version: [16.20.2]
```

Line 15
```yaml
- run: npm install -g yarn@1.22.21
```

File: yarn.lock [hunk 2]
```diff
@@ -8767,7 +8767,7 @@ strip-indent@^2.0.0:
 
 strip-indent@^3.0.0:
   version "3.0.0"
-  resolved "https://registry.yarnpkg.com/strip-indent/-/strip-indent-3.0.0.tgz#c32e1cee940b6b3432c771bc2c54bcce73cd3001"
+  resolved "https://registry.yarnpkg.com/strip-indent/-/strip-indent-3.0.0.tgz#c32e1cee940b6b3432c771bc2c54bcce73cd5000"
```
Comment (damonhook) [inline yarn.lock:8770]: This change was giving me a `Integrity check failed for "strip-indent"` when installing locally

**Review by damonhook — COMMENTED** [review]
Thank you for this.
I just have a few comments below

File: api/controllers/statsController.ts [hunk 3]
```diff
@@ -104,6 +105,37 @@ export class StatsController {
       .map((damage) => ({ damage, ...probabilities[damage] }));
   }
 
+  private buildInverseProbabilities(
+    probabilities: TMappedProbabilities,
+    fighterNames: string[]
+  ): TProbabilityResult[] {
```
[the remark this answers is not in the document]
Comment (Spoofardio) [reply api/controllers/statsController.ts:111]: ah missed these! might add a pre-commit hook In a new PR if I make more changes :)

File: yarn.lock (hunk 2 above)
Comment (Spoofardio) [reply yarn.lock]: ah, a mistake from changing the port locally for myself!

**Comment by Spoofardio** [comment]
@damonhook Thank you for getting back to me and the feedback! I made changes for each comment you made

**Review by damonhook — APPROVED** [review]

**Pull request merged**